src/Microsoft.AspNetCore.Mvc.Localization/Internal/MvcLocalizationServices.cs

```diff
@@ -29,7 +29,7 @@ public static class MvcLocalizationServices
         }
 
         // To enable unit testing only 'MVC' specific services
-        public static void AddMvcLocalizationServices(
+        public static void AddMvcViewLocalizationServices(
```

Comment:
I realize this is `.Internal` and thus not a breaking change. But, why change it?

Reply:
To make the intention clear and yeah I changed it because it was `Internal`